commands.go

```diff
@@ -653,6 +653,8 @@ func cmdRm(c *cli.Context) {
 	}
 	if isError {
 		log.Fatal("There was an error removing a machine. To force remove it, pass the -f option. Warning: this might leave it running on the provider.")
+	} else {
```

Comment:
You don't need this `else`.  You can simply do

```
if isError {
...
}

log.Println("Machine removed").
```

...as `log.Fatal` will exit immediately with a non-zero.